packages/rust-lexicase/Cargo.toml

```diff
@@ -1,8 +1,15 @@
 [package]
 name = "rust-lexicase"
 version = "0.1.0"
-edition = "2021"
+authors = { workspace = true }
+description = { workspace = true }
+documentation = { workspace = true }
+repository = { workspace = true }
+edition = { workspace = true }
 license = { workspace = true }
+categories = ["algorithms"]
+keywords = ["lexicase-selection", "genetic", "evolutionary-computation"]
```

Comment:
Probably want to mention the Python cross-language work somewhere here? (Or not, since this seriously needs to be moved.)